src/main/java/net/minecraftforge/event/entity/living/EnderManAngerEvent.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.world.entity.monster.EnderMan;
+import net.minecraft.world.entity.player.Player;
+import net.minecraftforge.eventbus.api.Cancelable;
+
+/**
+ * Event that is fired when an enderman checks if the player is looking at him.
+ * If cancelled the enderman does not get angry at the player.
```

Comment:
Proposing a more formal javadoc
```
/**
 * This event is fired when an Enderman detects that a player is looking at them.
 * It will not be fired if the detection is prevented by {@link IForgeItem#isEnderMask}
 * 
 * This event is {@link Cancelable}.
 * If this event is canceled, the Enderman will not target the player.
 * 
 * This event does not have a {@link Result}.
*/